src/main/java/com/targomo/client/api/geo/Geometry.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * 
+ */
+package com.targomo.client.api.geo;
+
+/**
+ * @author gideon 
+ *
+ */
+public interface Geometry extends Location {
```

Comment:
So there are now two Geometry classes? why do we need a new one?

Reply:
If I extend the existing geometry class I cannot create some of the things required by jackson to serialise/deserialise the class

Reply:
Changed name to `RoutingGeometry` to make things clearer